### DISCUSSION TITLE
Problem (New User) with Line to Satin.  Please help

### DISCUSSION THREAD

**Discussion by sdrose** [inkstitch/inkstitch#1792] [Q&A] [answered]
Hello all. 
I was reviewing the Satin Stitch from Bezier Curve (with a wide stroke, then Satin Tools -> Convert stroke to Satin.  I followed the Youtube video, but the lines are not converting properly to the Satin.  Was hoping someone would see the attached graphic and tell me either a step I missed or what my error is please.

Thank you
-dave
<img width="1112" alt="Ink Stitch Line to Satin Problem" src="https://user-images.githubusercontent.com/13631556/184945420-e8cd24e3-00d7-4ebf-920b-9e8f264f64ab.png">

**Comment by tyrosinase** [comment, marked as answer]
I think that's the bug where one side of the satin gets reversed, and instead of satin you get string art. Devs have been working on it though I don't know if there's a fix out yet.

The workaround is pretty easy though: Node-edit the shape, and select one side or the other and do a Path Reverse on it. (If you don't select just one side you'll reverse the whole path and nothing will effectively change.) What you want is for both rails to go the same direction. I always keep `Preferences > Tools > Nodes > Show path direction on outlines` turned on so I can see which way things will stitch out.

**Comment by sdrose** [comment]
Thankn you @tyrosinase!

Just for completeness, I wanted to add the graphics from this answer.  So, I created a single Bezier Curve Stroke, then thickened the stroke to 1.8mm, went to Extensions -> Ink/Stitch -> Tools:  Satin -> Convert Line to Satin.

Then after this command, the 2 rails with rungs are displayed on the Inkscape screen.  But clicking the Node Edit button brings up not rails to be reversed, but a line of nodes.  
<img width="1123" alt="Screen Shot 2022-08-16 at 3 08 38 PM" src="https://user-images.githubusercontent.com/13631556/184964692-a90ad727-71bf-4e2b-a1c5-514629b2c28e.png">
<img width="548" alt="Screen Shot 2022-08-16 at 3 21 47 PM" src="https://user-images.githubusercontent.com/13631556/184964696-90383c26-158c-416f-9132-7b6df5271bfb.png">

Clicking on a single   ending node, and choosing Path -> Reverse will then correct the Satin.
<img width="805" alt="Screen Shot 2022-08-16 at 3 13 17 PM 2" src="https://user-images.githubusercontent.com/13631556/184964698-937db486-a395-4619-8f4a-3aa99c7cd765.png">
<img width="1085" alt="Screen Shot 2022-08-16 at 3 23 16 PM" src="https://user-images.githubusercontent.com/13631556/184964704-96c60d97-01e9-497d-bb6f-eeb5698c932a.png">
  Then a command to trim a jump stitch can be added as necessary & hardware supported.

**Comment by tyrosinase** [comment]
Yeah, the node points can obscure the actual lines when you're in that mode, depending on your zoom level (and marker size, which I think can be adjusted in Preferences but I never do). But they're under there, and as you say you only have to select one or two for the Reverse to know what to work on.
